cmd/utils/flags.go

```diff
@@ -916,8 +916,9 @@ var (
 		Category: flags.RollupCategory,
 	}
 	RollupSuperchainUpgradesFlag = &cli.BoolFlag{
-		Name:     "beta.rollup.superchain-upgrades",
-		Usage:    "Beta feature: apply superchain-registry config changes to the local chain-configuration",
+		Name:     "rollup.superchain-upgrades",
+		Aliases:  []string{"beta.rollup.superchain-upgrades"},
+		Usage:    "Apply superchain-registry config changes to the local chain-configuration",
 		Category: flags.EthCategory,
```

Comment:
is that category actually correct?

Reply:
Oh that's the grouping in the help section, I think we should move it to the rollup config category, good catch.